sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -262,7 +265,13 @@ def send_typing_notification(
                 :dedent: 8
                 :caption: Send typing notification.
         """
-        return self._client.chat_thread.send_typing_notification(self._thread_id, **kwargs)
+
+        sender_display_name = kwargs.pop("sender_display_name", None)
+        send_typing_notification_request = SendTypingNotificationRequest(sender_display_name=sender_display_name)
+        return self._client.chat_thread.send_typing_notification(
+            chat_thread_id=self._thread_id,
+            send_typing_notification_request=send_typing_notification_request,
+            **kwargs)
```

Comment:
Is there a difference between:
```python
return self._client.chat_thread.send_typing_notification(
    chat_thread_id=self._thread_id,
    send_typing_notification_request=None,
    **kwargs)
```
And this:
```python
return self._client.chat_thread.send_typing_notification(
    chat_thread_id=self._thread_id,
    send_typing_notification_request=SendTypingNotificationRequest(sender_display_name=None),
    **kwargs)
```

Reply:
I think it's no. Since the sender display name is an optional param